### DISCUSSION TITLE
Nowhere is mentioned what the syntax is for calling child components?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [vue-vine/vue-vine#161] [Q&A] [answered]
Nowhere is the syntax mentioned for using children, so components inside other vine templates?

```js
function A() {
 return vine`<p>hello from A</p>`
}

function B() {
 return vine`<div>${A()}</div>`
}
```

Is just calling the function correct?

**Comment by ShenQingchuan** [comment, marked as answer]
No @ivanjeremic , please check what we mentioned in our docs: https://vue-vine.dev/specification/overview.html#template

> Using expression interpolation in template is forbidden.
> 
> Because the whole tagged template string is a raw Vue template. Vine compiler will transfer it to `@vue/compiler-dom` to compile it to render function in the end.

The correct way to reference a child component it just the same as what you did in Vue SFC.

```ts
function A() {
  return vine`<p>hello from A</p>`
}

function B() {
  return vine`
    <div>hello from B</div>

    <!-- ↓ Using A as a Vue component tag here -->
    <A /> 
  `
}
```
